.travis.yml

```diff
@@ -12,13 +12,10 @@ branches:
     - travis
 
 rvm:
-  - 1.8.7
```

Comment:
Ruby 1.8.7-p370 was released Jun 29. 2012, almost four years ago at this point.

Are people still using it?